osg-koji

```diff
@@ -253,8 +254,13 @@ def create_client_symlink_to_proxy(new_client_cert_path):
 def setup_koji_client_cert(user_cert, user_key):
     """Create or copy the client cert file (if needed)."""
     new_client_cert_path = os.path.join(OSG_KOJI_USER_CONFIG_DIR, CLIENT_CERT_FILE)
-    if (os.path.exists(new_client_cert_path) and
-            not ask_yn("Client cert file '" + new_client_cert_path + "' already exists. Recreate it?")):
+    if (os.path.lexists(new_client_cert_path) and
+            not ask_yn("""
+Client cert file '%s' already exists.
```

Comment:
The program already displays a prompt of `[y/n]`, so 'Enter yes ("y")' is superfluous, but I'll tweak the rest of it.